skorch/callbacks.py

```diff
@@ -365,3 +365,79 @@ def on_epoch_end(self, net, **kwargs):
         self._sink(tabulated.rsplit('\n', 1)[-1], verbose)
         if self.sink is print:
             sys.stdout.flush()
+
+
+class Checkpoint(Callback):
+    """Save the model during training if the given metric improved.
+
+    This callback works by default in conjunction with the validation
+    scoring callback since it creates a `valid_loss_best` value
+    in the history which the callback uses to determine if this
+    epoch is save-worthy.
+
+    You can also specify your own metric to monitor or supply a
+    callback that dynamically evaluates whether the model should
+    be saved in this epoch.
+
+    Example:
+
+    >>> net = MyNet(callbacks=[Checkpoint()])
+    >>> net.fit(X, y)
+
+    Example using custom monitor:
```

Comment:
...that only saves the model when both train and valid loss improved.

Reply:
Yes. if you have a better example I will replace this one. However, for the sake of showing how to use this callback this example suffices IMO.

Reply:
No, I just meant that the sentence should be extended to make it more transparent what happens.